src/lib.rs

```diff
@@ -90,7 +90,7 @@
 //!
 //! or use the attribute macros.
 //!
-//! ```
+//! ```ignore
```

Comment:
Is there a reason you ignore this doc test?

Reply:
yeah, the propper way to run tests seems to be via `ci.sh test`, not `cargo test` directly. `cargo test` fail, so i copied the `ignore` from another example.

Thought about making another PR, that removes the `ignore`s on code examples.

Reply:
Ah I remember, then its fine